content/docs/1.6.0/advanced-resources/security/volume-encryption.md

```diff
@@ -104,4 +102,5 @@ transparently used so no additional actions are needed from the user.
 Longhorn supports offline [expansion](../../../volumes-and-nodes/expansion) for encrypted volumes.
 
 # History
-Available since v1.2.0 [#1859](https://github.com/longhorn/longhorn/issues/1859)
+- Support Encryption for Volume in Filesystem Mode since v1.2.0 [#1859](https://github.com/longhorn/longhorn/issues/1859)
```

Comment:
Please use the following wording:

Encryption of volumes in Filesystem mode available starting v1.2.0